### PR TITLE
sys-kernel/pf-sources: bump to 5.2-pf1 && add maintainer

### PR DESCRIPTION
**Pull request by juippis**

### CODE REVIEW

File: sys-kernel/pf-sources/pf-sources-5.2_p1.ebuild [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+IUSE=""
+
+# No 'experimental' USE flag provided, but we still want to use genpatches
+K_EXP_GENPATCHES_NOUSE="1"
+
+# No reason to bump this number unless something new gets included in genpatches,
+# in that case we can manually remove the linux kernel patches from genpatches.
+K_GENPATCHES_VER="1"
+
+K_NOSETEXTRAVERSION="don't_set_it"
```
Comment (mgorny) [inline sys-kernel/pf-sources/pf-sources-5.2_p1.ebuild]: Wouldn't simple `1` work?

File: sys-kernel/pf-sources/pf-sources-5.2_p1.ebuild [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	ewarn
+	kernel-2_pkg_setup
+}
+
+src_prepare() {
+	default
+	kernel-2_src_prepare
+}
+
+pkg_postrm() {
```
Comment (mgorny) [inline sys-kernel/pf-sources/pf-sources-5.2_p1.ebuild]: I don't think you need to redeclare it, do you?

File: sys-kernel/pf-sources/pf-sources-5.2_p1.ebuild (hunk 1 above)
Comment (juippis) [reply sys-kernel/pf-sources/pf-sources-5.2_p1.ebuild]: It should. Remnants of the old ebuild...

**Comment by juippis** [comment]
No idea what I can do to fix buildbot status. Seems like it cant handle rebases on these PRs.

**Comment by mjeveritt** [comment]
Oh dear - this is rather unhelpful .. and I'm not sure *why* its failing on this PR ..
Leave it with me, and I'll try to get a fix up.

**Comment by mjeveritt** [comment]
Seems to be working fine now.
Please re-trigger the checks, and it should pass.
See http://kernel1.amd64.dev.gentoo.org:8010/#/builders/16/builds/1811 for repeat build.

**Comment by mjeveritt** [comment]
> Seems to be working fine now.
> Please re-trigger the checks, and it should pass.
> See http://kernel1.amd64.dev.gentoo.org:8010/#/builders/16/builds/1811 for repeat build.

Scratch that, Github has figured there's been a successful re-attempt.
:+1: from us, and hopefully welcome aboard team kernel!

**Pull request closed without merging**